spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/order'
+require './lib/takeaway'
+
+describe Order do
+  subject(:order) {Order.new}
+  let(:dish1) { double :'Chow mein', dish: 'chow mein', price: '3' }
```

Comment:
Great use of double but can also be used in a let statment to do pricing for multiple lines. Helps keep code DRY.